schema/release-schema.json

```
@@ -1736,7 +1736,274 @@
             "string",
             "null"
           ],
-          "description": "The country name. For example, United States."
+          "description": "The country name. For example, United States.",
+          "deprecated": {
+            "description": "This field is deprecated, because OCDS isn't the place to annotate a country with additional information like its human-readable name. ISO3166-1 alpha-2 country codes can be provided in `.country`.",
```

Reworded to describe the value the new field brings.

```suggestion
            "description": "This field is deprecated in favor of `country`, to promote standardized country codes instead of non-standardized country names.",
```